src/EditorFeatures/CSharpTest/CodeActions/InvertIf/InvertIfTests.Elseless.cs

```diff
@@ -0,0 +1,595 @@
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void M()
+    {
+        while (c)
+        {
+            if (c)
+            {
+                [||]if (c)
+                {
+                    continue;
+                }
+                if (c())
+                    return;
+            }
+        }
+    }
+}",
+@"class C
+{
+    void M()
+    {
+        while (c)
+        {
+            if (c)
+            {
+                [||]if (!c)
+                {
+                    if (c())
+                        return;
+                }
+            }
+        }
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInvertIf)]
+        public async Task TestElseless13()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void M()
+    {
+        while (c)
+        {
+            {
+                [||]if (c)
+                {
+                    continue;
+                }
+                if (c())
+                    return;
+            }
+        }
+    }
+}",
+@"class C
+{
+    void M()
+    {
+        while (c)
+        {
+            {
+                [||]if (!c)
+                {
+                    if (c())
+                        return;
+                }
+            }
+        }
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInvertIf)]
+        public async Task TestElseless14()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void M()
+    {
+        [||]if (c)
+        {
+            f();
+        }
+        g();
+        g();
+    }
+}",
+@"class C
+{
+    void M()
+    {
+        if (!c)
+        {
+        }
+        else
+        {
+            f();
+        }
+        g();
+        g();
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInvertIf)]
+        public async Task TestElseless15()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    bool M()
+    {
+        if (c)
+        {
+            [||]if (c)
+            {
+                f();
+            }
+            g();
+        }
+        return false;
+    }
+}",
+@"class C
+{
+    bool M()
+    {
+        if (c)
+        {
+            if (!c)
+            {
+            }
+            else
+            {
+                f();
+            }
+            g();
+        }
+        return false;
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInvertIf)]
+        public async Task TestElseless16()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void M()
+    {
+        [||]if (c)
+        {
+            f();
+        }
+
+        g();
+        g();
+    }
+}",
+@"class C
+{
+    void M()
+    {
+        if (!c)
+        {
+        }
+        else
+        {
+            f();
+        }
+
+        g();
+        g();
+    }
+}");
+        }
+    }
+}
```

Comment:
I didn't see any tests for the `IfWithoutElse_WithNegatedCondition` case.  Also, since there are so many different configurations identified in `InvertIfStyle`, it would be helpful to name the tests according to which one they're testing.